### ISSUE TITLE
Python: using uid instead of id

### ISSUE THREAD

**Issue by jeertmans** [SnaKyEyeS/ADE-Scheduler#310]
*Confusion of naming*

`id` is a Python builtin function and I think we should, in the future, change the naming all `id` variables to `uid` to avoid confusion.

*Changes to do*

This currently concerns the `Schedule` objects.
Changes are mostly in Python code but one should be aware to:
1. verify that JS files have their endpoints updated accordingly if needed (i.e. change id to uid)
2. verify that you old objects stored in the database to still work (would be sad that object.uid does not exist when deployed on production)

**Comment by SnaKyEyeS** [comment]
I disagree with the idea to trade the `id` variable name with `uid`: it is just transforming one problem in another (even though `uid` is not a built-in funciton per se, it is still widely used).
However, a better idea would be to replace `id` by `calendar_id` for a calendar's ID or `course_id` for a course's ID and so on.

That being said, I'm not exactly sure this is worth it... if you got time to lose, be my guest ! :D

**Comment by SnaKyEyeS** [comment]
OK I was wrong, we're using the function `uuid()`, not `uid()` but my point about a confusing meaning of the variable stands.
